lib/monkey-mailer/adapters/mandrilapi.rb

```diff
@@ -59,6 +56,27 @@ def send_email(email)
         raise MonkeyMailer::DeliverError.new("Mandril response.code not equal to 200") unless response.code.to_i == 200
         puts "Response #{response.code} #{response.message}: #{response.body}"
       end
+
+      def parse_recipients(recipients_list, names_list = "", type = :to)
+        recipients = recipients_list.split(",")
+        names      = names_list.to_s.split(",")
```

Comment:
If I'm not mistaken, the `names_list` will be a string always unless is miss-used, so I wont be `nil` in real use cases, do you think we can get rid of the `to_s`?